motif/testdata/expected.jasparPFM.consensus.fa

```diff
@@ -5,17 +5,17 @@ TGCGTG
 >Ddit3::Cebpa
 AGATGCAATCCC
 >Mecom
-AAGATAAGATAAGA
+AAGATAAGATAATA
```

Comment:
I didnt dive too far into the code but maybe double check this one with @rimangan. When I checked on a fork from your branch, I got something like this:
```text
diff
AAGATAAGATAAGA
AAGATAAGATAATA
--- FAIL: TestConsensusSequences (0.01s)
    convert_test.go:90: Error in Consensus Sequences. Output not as expected.
```